app/models/metasploit/credential/nonreplayable_hash.rb

```diff
@@ -39,88 +92,193 @@ class Metasploit::Credential::NonreplayableHash < Metasploit::Credential::Passwo
     hmac-sha384
     hmac-sha512
     hmailserver
+    hsrp
+    ike
     ipb2
+    itunes-backup
+    iwork
     keepass
     keychain
+    keyring
+    keystore
+    known_hosts
     krb4
     krb5
+    krb5-17
+    krb5-18
+    krb5-3
+    krb5asrep
+    krb5pa-md5
+    krb5pa-sha1
+    krb5tgs
```

Comment:
I'm not really sure what the intended semantics of 'non replayable' is when storing these types of creds; But for visibility the Kerberos tickets that are used to generate these hashes are not replayable, whilst the tickets themselves generally last for a day or so - unless they're golden tickets.

Reply:
I believe 'non replayable' refers to if the hash would be usable to create a new authenticated session with the original auth provider, in the case of kerberos tickets I think the ticket itself may be 'non replayable' as it cannot be used to negotiate a new session after it expires, even if it is usable until the expiration with services that will validate it against the original provider.